src/docs_precompile/make.jl

```diff
@@ -0,0 +1,11 @@
+using Documenter, Logging
+
+with_logger(NullLogger()) do
+makedocs(
+    sitename = "TestPkg",
+    pages = Any[
+        "Home" => "index.md",
+    ],
+    build = mktempdir()
+)
```

Comment:
Could we also add another `makedocs` call here with `format = Documenter.LaTeX(platform = "none")` to catch the LaTeX-related methods?

Reply:
Probably yes, but I'm not sure how common it is to generate latex docs so the benefit is perhaps significantly smaller.

Reply:
That is true, but would it add much to the precompilation time? Also, I think we could also just set `format = [HTML(), LaTeX(...)]`, and then we won't have to re-run the other build steps.

Reply:
It looks like adding LaTeX to adds about 50% to the precompilation / first-run time, which is not that great. So yeah, I don't think it makes sense to add it.